### PR TITLE
Issue 2196 fix unittest warnings r3.3

### PR DESCRIPTION
**Pull request by syneva-runyan**
Issue:

## What I did

Cleaned up warnings in unit tests

### CODE REVIEW

File: lib/ui/src/modules/shortcuts/actions/shortcuts.test.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ describe('manager.shortcuts.actions.shortcuts', () => {
 
     test('should warn about deprecated option names', () => {
       const clientStore = new MockClientStore();
-      const spy = jest.spyOn(console, 'warn');
+      const spy = jest.spyOn(console, 'warn').mockImplementation(() => {});
```
Comment (dangreenisrael) [inline lib/ui/src/modules/shortcuts/actions/shortcuts.test.js]: Is this meant to swallow prop type warnings instead of failing?

Comment (syneva-runyan) [reply lib/ui/src/modules/shortcuts/actions/shortcuts.test.js]: No, the test makes sure that the warning is called, which is still checked.  By adding the mock implementation, the console.warnings dont get printing in the console for the duration of the specific test

Comment (dangreenisrael) [reply lib/ui/src/modules/shortcuts/actions/shortcuts.test.js]: Ah, awesome 😀

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,8 @@
 
 #### Bug Fixes
 
+- Cleaned up unit test warnings [#2196](https://github.com/storybooks/storybook/pull/2381)
```
Comment (Hypnosphi) [inline CHANGELOG.md]: You shouldn't edit this file unless you do a release

**Review by Hypnosphi — CHANGES_REQUESTED** [review]
please uncomment those lines: https://github.com/storybooks/storybook/blob/release/3.3/scripts/jest.init.js#L17-L22

**Comment by syneva-runyan** [comment]
@Hypnosphi changes made

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**